src/collection/collection.go

```diff
@@ -159,15 +160,19 @@ func buildSchemaListForDatabase(con *connection.PGSQLConnection) (SchemaList, er
 
 	for index, row := range dataModel {
 		if !row.SchemaName.Valid || !row.TableName.Valid {
-			log.Error("Query responded with a null schema name or table name. Skipping row %d", index)
+			if row.IndexName.Valid {
+				log.Debug("Skipping Index %s. Schema name or Table name null. [Schema: %s / Table %s]", row.IndexName.String)
```

Comment:
I see three %s and just one arg, is it a mistake?

Does it make sense to print the boolean (es: row.SchemaName.Valid) instead of a possible null string?

Reply:
🤦 fixed